### ISSUE TITLE
Be careful

### ISSUE THREAD

**Issue by infosergej** [lReDragol/OnceHuman_PhotoChanger#4]
The developers have taken it on ) 

![image](https://github.com/user-attachments/assets/27ffc43e-1bd9-422d-9613-c49876484e18)

**Comment by lReDragol** [comment]
> The developers have taken it on )
> 
> ![image](https://private-user-images.githubusercontent.com/48388489/365776513-27ffc43e-1bd9-422d-9613-c49876484e18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU5NjY1MTgsIm5iZiI6MTcyNTk2NjIxOCwicGF0aCI6Ii80ODM4ODQ4OS8zNjU3NzY1MTMtMjdmZmM0M2UtMWJkOS00MjJkLTk2MTMtYzQ5ODc2NDg0ZTE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEwVDExMDMzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1Y2E2YWEwNzlkMTc4NmFjMDg5N2ZjMmVhZDdlZjMwMjMzNDg4ZGE4YTJkNTFkZTc3MzJlNzAxYzU3NGY4MzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FH84amfFqp4lhxbz4uXN6o8YFXLk7s2RQh0NELyCzoM)

![image](https://github.com/user-attachments/assets/a0a9166a-f07a-47a9-bf45-3903cf55b79a)
I got such a quibble last night, too).
I was very amused by the phrase that 
“modify local image resources, which has led to disruptions in other players' gaming experiences.”!😂😂😂😂
I'm currently trying to figure out how the system was able to detect the photo spoofing. 
(But I think that the administration just manually checks the content of the players' album photos sometimes and in this case there is no way to bypass it).
So for those who have not received this message, try to make as many photos as possible in 1 day for future use and not to get blocked.
As soon as I get more information on this incident I will post a comment. If anyone knows anything or has any suggestions, you can write me in telegram @l_reDrago_l

**Comment by infosergej** [comment]
I'm sure that there was a manual check, because on our server was asked to check cheaters before the GWG, the rest of the group did not receive a letter of happiness :D

**Comment by fieelk** [comment]
the photo I'm adding is not being verified. Writes "The image could not be verified, viewing is not possible"
Don't know how to fix it? What am I doing wrong?

**Comment by lReDragol** [comment]
> the photo I'm adding is not being verified. Writes "The image could not be verified, viewing is not possible" Don't know how to fix it? What am I doing wrong?
restart the program and log in to the server

**Comment by fieelk** [comment]
> > фотография, которую я добавляю, не проверяется. Пишет «Изображение не удалось проверить, просмотр невозможен». Не знаю, как это исправить? Что я делаю не так? перезапустите программу и войдите на сервер

No, it doesn't help, the photo is checked for a long time, then he writes the same thing(((

**Comment by infosergej** [comment]
> > > фотография, которую я добавляю, не проверяется. Пишет «Изображение не удалось проверить, просмотр невозможен». Не знаю, как это исправить? Что я делаю не так? перезапустите программу и войдите на сервер
> 
> 
> 
> No, it doesn't help, the photo is checked for a long time, then he writes the same thing(((

I encountered such a problem a couple of times, but after restarting to the server, everything works.

**Comment by fieelk** [comment]
> > > > Фотография, которую я добавляю, не проверяется. Пишет «Изображение не удалось проверить, просмотр невозможен». Не знаю, как это исправить? Что я делаю не так? перезапустите программу и войдите на сервер
> > 
> > 
> > Нет, это не помогает, фото долго проверяется, а потом он пишет то же самое(((
> 
> Я сталкивался с такой проблемой пару раз, но после перезапуска сервера всё работало.

I think I understood that it does not skip all images, as if it depends on the contents of the image, everything with a rating of 18+ does not pass verification(

**Comment by fieelk** [comment]
> > > > фотография, которую я добавляю, не проверяется. Пишет «Изображение не удалось проверить, просмотр невозможен». Не знаю, как это исправить? Что я делаю не так? перезапустите программу и войдите на сервер
> > 
> > 
> > Нет, это не помогает, фото долго проверяется, потом он пишет одно и то же(((
> 
> Я пару раз сталкивался с такой проблемой, но после перезагрузки на сервер все работает.

Can you please describe your actions after which an image appears in your frame? it is in the game, not in the program
